hokusai/development.yml

```diff
@@ -2,9 +2,9 @@
 version: "2"
 services:
   horizon:
+    env_file: ../.env
```

Comment:
One problem we've had in other projects is that `hokusai dev` will _fail_ if this file isn't found. If you've found this useful, you might want to add a step to the set-up described in the README.